coral/src/domain/topic/index.ts

```diff
@@ -1,5 +1,5 @@
-import { Topic } from "src/domain/topic/topic-types";
-import { getTopics } from "src/domain/topic/topic-api";
+import { getTopicNames, getTopics } from "src/domain/topic/topic-api";
```

Comment:
Yes, I thought this was annoying ^^' I usually use the"Organize import" feature of VScode when I have missing of unusaed imports, which often rearranges imports in this way. I can stop doing that if it is annoying ^^'

Reply:
Oh, it's not. But I think it may be a good idea to have a rule there, so all imports are ordered the same for everyone. Something for a housekeeping issue!